My Spotify/My Spotify/Modules/ViewArtistUserProfile/Coordinator/ViewArtistCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ViewArtistCoordinator.swift
+//  My Spotify
+//
+//  Created by Devarsh Bhalara on 10/07/23.
+//
+
+import Foundation
+import UIKit
+
+class ViewArtistCoordinator: Coordinator {
+    var childCoordinators: [Coordinator] = []
+    var navigationController: UINavigationController
+    var artistId: String
+    
+    init(navigationController: UINavigationController, artistId: String) {
+        self.navigationController = navigationController
+        self.artistId = artistId
+    }
+    
+    func start() {
+        let vc = ArtistProfileVC.instantiate(from: .viewPorfile)
+        vc.coordinator = self
+        vc.artistId = self.artistId
```

Comment:
remove `self` keep code concise 
```suggestion
        vc.artistId = artistId
```